### PR TITLE
feat: hotspot implementation

### PR DESCRIPTION
**Pull request by sergiulucaci**
This PR includes [hotspot functionality](https://app.clubhouse.io/keydatasystems/story/1193/hotspot-rendering-ui?team_id=6). I still have to add small changes (Abe found some issues), but let me know what you think about it, Ed.

Take note that this PR doesn't include unit tests nor documentation yet.

### CODE REVIEW

**Comment by edeustace** [comment]
so wait for tests?

File: packages/hotspot/src/hotspot/rectangle.jsx [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+          draggable={false}
+          stroke={outlineColorParsed}
+          strokeWidth={outlineWidth}
+          onMouseLeave={this.handleMouseLeave}
+          onMouseEnter={this.handleMouseEnter}
+          x={x}
+          y={y}
+        />
+
+        {isEvaluateMode ? (
```
Comment (edeustace) [inline packages/hotspot/src/hotspot/rectangle.jsx]: you don't need 'evaluate' in your component, often just disabled and has correct response info is enough

Comment (sergiulucaci) [reply packages/hotspot/src/hotspot/rectangle.jsx]: Not sure if I get it, are you saying that `isEvaluateMode ` can be replaced with a combination between disabled and has correct response in all occurrences from `rectangle.jsx`? If so, what if there's no correct response?

Comment (edeustace) [reply packages/hotspot/src/hotspot/rectangle.jsx]: you can leave it - the point is that the controller will prepare the ui model for you, so pie specific attributes aren't needed if you want to make your logic simpler.

**Pull request merged**